ts/Series/GeoHeatmap/GeoHeatmapSeries.ts

```diff
@@ -0,0 +1,297 @@
+/* *
+ *
+ *  (c) 2010-2023 Highsoft AS
+ *
+ *  Authors: Magdalena Gut, Piotr Madej
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type GeoHeatmapSeriesOptions from './GeoHeatmapSeriesOptions.js';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+import GeoHeatmapPoint from './GeoHeatmapPoint.js';
+
+const {
+    seriesTypes: {
+        map: MapSeries
+    }
+} = SeriesRegistry;
+
+import U from '../../Core/Utilities.js';
+
+const {
+    extend,
+    merge
+} = U;
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+/**
+ * The Geo Heatmap series type.
+ *
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.geoheatmap
+ *
+ * @augments Highcharts.Series
+ */
+
+class GeoHeatmapSeries extends MapSeries {
+
+    /* *
+     *
+     *  Static Properties
+     *
+     * */
+
+    /**
+     * A `geoheatmap` series is a variety of heatmap series, composed into
+     * the map projection, where the units are expressed in the latitude
+     * and longitude, and individual values contained in a matrix are
+     * represented as colors.
+     *
+     * @sample maps/demo/geoheatmap/
+     *         GeoHeatmap Chart on the Orthographic Projection
+     * @sample maps/demo/geoheatmap-equalearth/
+     *         GeoHeatmap Chart on the Equal Earth Projection
+     *
+     * @extends      plotOptions.map
+     * @since        next
+     * @product      highmaps
+     * @excluding    allAreas, dragDrop, findNearestPointBy, geometry, joinBy, negativeColor, onPoint,
```

Comment:
I think, we should exclude more API properties, which are not working for `GeoHeatMap` series, e.g. `boostBlending`, `boostThreshold` (boost is not even working in maps, maybe it should also be excluded in `MapSeries`?